src/main/java/org/apache/maven/plugins/assembly/archive/task/AddDependencySetsTask.java

```diff
@@ -183,7 +185,12 @@ void addDependencySet( final DependencySet dependencySet, final Archiver archive
 
     private ProjectBuildingRequest getProjectBuildingRequest( AssemblerConfigurationSource configSource )
     {
-        return configSource.getMavenSession().getProjectBuildingRequest();
+        MavenSession mavenSession = configSource.getMavenSession();
+        return new DefaultProjectBuildingRequest()
+                .setRepositorySession( mavenSession.getRepositorySession() )
+                .setSystemProperties( mavenSession.getSystemProperties() )
+                .setUserProperties( mavenSession.getUserProperties() )
+                .setProcessPlugins( false );
     }
```

Comment:
Why not reusing the existing `ProjectBuildingRequest` ? What does that actually change ?
Shouldn't that be something like:
```
return  new DefaultProjectBuildingRequest( configSource.getMavenSession().getProjectBuildingRequest() )
      .setProcessPlugins( false );
```
I fear not reusing the existing will loose remote repositories and other settings, not sure though.